test/k8s-integration/main.go

```diff
@@ -344,6 +367,21 @@ func generateTestSkip(normalizedVersion string) string {
 	return skipString
 }
 
+// Helper function to generate skip string when using GKE managed PD CSI driver.
+func generateGKEBasedTestSkip(clusterVersion string) string {
+	skipString := "\\[Disruptive\\]|\\[Serial\\]"
+	curVer := MustParseVersion(clusterVersion)
+	if (*curVer).IsLessThan(MustParseVersion("1.17.0")) {
+		skipString = skipString + "|volumeMode\\sshould\\snot\\smount\\s/\\smap\\sunused\\svolumes\\sin\\sa\\spod"
+	}
```

Comment:
@msau42  is this the right version to skip the mount tests?

Reply:
The fix was in 1.16.0, it should be < 1.16.0. 

Alternatively, can we share the same skip function?

Reply:
Maybe run the cluster version through getNormalizedVersion, apply generateTestSkip, and then append additional skips based on the full version?

Reply:
@msau42  Its tricky to share the same skipstring, since that works on major.minor version only. However, for using GKE managed driver, say snapshots, we need to check the full version 1.18.3.
Perhaps, using both generateTestSkip, and additional filters based on GKE full version is the way to go as @mattcary  suggested.

Reply:
Can we normalize the "x.y" version to "x.y.z" so it'll work with the new "lessThan" checks?

Reply:
does x.y normalize to x.y.0?